### DISCUSSION TITLE
Why not GPL ?

### DISCUSSION THREAD

**Discussion by adamency** [nushell/nushell#8126] [General]
I was just curious about the licensing choice of the project, as I would have felt that this kind of project would be more protected with a license like the GPL in order to prevent a private company from exploiting the project by creating a more powerful and featureful close-source version of `nushell` that would supersede the open-source project in popularity, while MIT is more relevant for backbone projects like libraries and drivers to enable mass usage as they will likely get embedded in an executable and GPL would hinder this.

Basically I feel that Nushell isn't much helped by MIT's terms and wouldn't lose a lot using GPL, while it would gain a huge protection.

Just wanted to hear the opinions of the maintainers :).
